tests/pytest_runner.py

```diff
@@ -1,17 +1,29 @@
+import glob
 import os
 
 import pytest
 
-from yt.config import ytcfg
-
 if __name__ == "__main__":
     os.environ["OMP_NUM_THREADS"] = "1"
     pytest_args = [
-        f"--local-dir={os.path.join(ytcfg.get('yt', 'test_data_dir'), 'answers')}",
-        "-c=pytest_answer.ini",
-        "--junitxml=unittests.xml",
-        "--answer-big-data",
-        f"-n {int(os.environ.get('NUM_WORKERS', 6))}",
+        "-s",
+        "-v",
+        "-rsfE",
```

Comment:
if this is what I think it is (4 arguments for on dash), I'd advise they be spliced, one per line... in which case the `s` is redundant. This will make the results of git blame much cleaner next time one of them is updated. Also it allows to add separate comments.

Reply:
it's not.
```
  -r chars              show extra test summary info as specified by chars:
                        (f)ailed, (E)rror, (s)kipped, (x)failed, (X)passed,
                        (p)assed, (P)assed with output, (a)ll except passed
                        (p/P), or (A)ll. (w)arnings are enabled by default (see
                        --disable-warnings), 'N' can be used to reset the list.
                        (default: 'fE').
```

Reply:
Fair enough. Can this be hinted to in a comment please ?

Reply:
added a comment